osu.Game.Tests/Visual/SongSelect/TestSceneBeatmapInfoWedge.cs

```diff
@@ -105,7 +107,8 @@ private void testBeatmapLabels(Ruleset ruleset)
             AddAssert("check version", () => infoWedge.Info.VersionLabel.Text == $"{ruleset.ShortName}Version");
             AddAssert("check title", () => infoWedge.Info.TitleLabel.Text == $"{ruleset.ShortName}Source — {ruleset.ShortName}Title");
             AddAssert("check artist", () => infoWedge.Info.ArtistLabel.Text == $"{ruleset.ShortName}Artist");
-            AddAssert("check author", () => infoWedge.Info.MapperContainer.Children.OfType<OsuSpriteText>().Any(s => s.Text == $"{ruleset.ShortName}Author"));
+            AddAssert("check author", () =>
+                infoWedge.Info.MapperContainer.ChildrenOfType<LinkFlowContainer>().Single().ChildrenOfType<OsuSpriteText>().Any(s => s.Text == $"{ruleset.ShortName}Author"));
```

Comment:
Suppose this one's my brainfart, but can simplify to
```suggestion
                infoWedge.Info.MapperContainer.ChildrenOfType<OsuSpriteText>().Any(s => s.Text == $"{ruleset.ShortName}Author"));
```